producers/src/unit-test/java/uk/gov/moj/sdt/producers/sdtws/SdtInternalEndpointPortTypeTest.java

```diff
@@ -61,53 +67,50 @@ public class SdtInternalEndpointPortTypeTest extends AbstractSdtUnitTestBase {
     /**
      * Mocked IWsUpdateItemHandler instance.
      */
+    @Mock
     private IWsUpdateItemHandler mockUpdateItemHandler;
 
     /**
      * Set up common for all tests.
      */
-    @Before
+    @BeforeEach
+    @Override
     public void setUp() {
-        portType = new SdtInternalEndpointPortType();
+        MockitoAnnotations.openMocks(this);
 
-        mockUpdateItemHandler = EasyMock.createMock(IWsUpdateItemHandler.class);
+        portType = new SdtInternalEndpointPortType();
         portType.setUpdateItemHandler(mockUpdateItemHandler);
-
     }
 
     /**
      * Test update item method completes successfully.
      */
     @Test
-    public void testUpdateItemSuccess() {
-        EasyMock.expect(mockUpdateItemHandler.updateItem(EasyMock.anyObject(UpdateRequestType.class))).andReturn(
-                createUpdateResponse());
-        EasyMock.replay(mockUpdateItemHandler);
+    void testUpdateItemSuccess() {
+        when(mockUpdateItemHandler.updateItem(any())).thenReturn(createUpdateResponse());
 
         final UpdateResponseType response = portType.updateItem(createUpdateRequest());
-        EasyMock.verify(mockUpdateItemHandler);
-        assertNotNull("Response expected", response);
+        verify(mockUpdateItemHandler).updateItem(any());
+        assertNotNull(response, "Response expected");
     }
 
     /**
      * Test update item method handles exceptions successfully.
      */
     @Test
-    public void testUpdateItemException() {
-        EasyMock.expect(mockUpdateItemHandler.updateItem(EasyMock.anyObject(UpdateRequestType.class))).andThrow(
-                new RuntimeException("test"));
-        EasyMock.replay(mockUpdateItemHandler);
+    void testUpdateItemException() {
+        when(mockUpdateItemHandler.updateItem(any())).thenThrow(new RuntimeException("test"));

```

Comment:
Previous version used EasyMock.anyObject(UpdateRequestType.class) for updateItem() parameter. Should this use any(UpdateRequestType.class) rather than just any()?